examples/inference-deployments/instructor/instructor_xl.yaml

```diff
@@ -8,8 +8,8 @@ command: |
   PYTHONPATH=.. uvicorn serve:app --host 0.0.0.0 --port 8080
 integrations:
 - integration_type: git_repo
-  git_repo: dakinggg/examples
-  git_branch: restructure
+  git_repo: mosaicml/examples
```

Comment:
@dakinggg not sure if you intended to keep your branch here?

Reply:
I did not 😬